cloud/scope/clients.go

```diff
@@ -158,19 +166,22 @@ func createClients(region string, oCIAuthConfigProvider common.ConfigurationProv
 	}, err
 }
 
-func createVncClient(region string, ociAuthConfigProvider common.ConfigurationProvider, logger *logr.Logger) (*core.VirtualNetworkClient, error) {
+func (c *ClientProvider) createVncClient(region string, ociAuthConfigProvider common.ConfigurationProvider, logger *logr.Logger) (*core.VirtualNetworkClient, error) {
 	vcnClient, err := core.NewVirtualNetworkClientWithConfigurationProvider(ociAuthConfigProvider)
 	if err != nil {
 		logger.Error(err, "unable to create OCI VCN Client")
 		return nil, err
 	}
 	vcnClient.SetRegion(region)
+	if c.ociClientHostUrls != nil && c.ociClientHostUrls.VCNClientHost != nil {
+		vcnClient.Host = *c.ociClientHostUrls.VCNClientHost
+	}
 	vcnClient.Interceptor = setVersionHeader()
```

Comment:
looks like answer to mys question on where client initialization will throw an error is that it will not, but we can verify.

Reply:
Would you expect/like it to fail and error?

Reply:
it should not, for example in some edge environments, there may not be oke endpoint, in that case, the entire init should not fail